### PR TITLE
Fix for Issue #75

### PR DESCRIPTION
**Pull request by jonboiser**
An attempt at fixing #75

New specs pass and no known regressions, but could use review, since I’m not familiar with these grammar files. 

Thanks for the pointer on running the specs. Is there a way I can symlink my copy of the library with Atom just to see the syntax highlighting in action?

### CODE REVIEW

**Comment by 50Wliu** [comment]
> Is there a way I can symlink my copy of the library with Atom just to see the syntax highlighting in action?

`apm link` inside of the `language-coffee-script` folder.

File: grammars/coffeescript.cson [hunk 1]
```diff
@@ -229,7 +229,7 @@
     'name': 'meta.function.coffee'
   }
   {
-    'match': '@?\\b(?!class|subclass|extends|decodeURI(Component)?|encodeURI(Component)?|eval|parse(Float|Int)|require)\\w+(\\s+(?!(of|in|then|is|isnt|and|or|for|else|when|not|if)\\s)(?=(\\w+|\\->|\\-\\d|\\[|\\{|\"|\'))|(?=\\())'
+    'match': '@?\\b(?!class|subclass|extends|decodeURI(Component)?|encodeURI(Component)?|eval|parse(Float|Int)|require)\\w+(\\s+(?!(of|in|then|is|isnt|and|or|for|else|when|not|if)\\s)(?=(\\@?\\w+|\\->|\\-\\d|\\[|\\{|\"|\'))|(?=\\())'
```
Comment (50Wliu) [inline grammars/coffeescript.cson]: I don't think the `@` needs to be escaped.

Comment (MaximSokolov) [inline grammars/coffeescript.cson]: `@` shouldn't be escaped

Comment (jonboiser) [inline grammars/coffeescript.cson]: Yeah, it works without it.

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**